src/components/TextEditor/components/TextEditor.js

```diff
@@ -17,25 +15,12 @@ import { SketchThumbnailArray } from "../../Sketches/constants";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Redirect } from "react-router-dom";
 
-let CodeMirror = null;
-if (typeof window !== "undefined" && typeof window.navigator !== "undefined") {
-  // import {Controlled as CodeMirror} from 'react-codemirror2'
-  CodeMirror = require("react-codemirror2").Controlled;
-  require("codemirror/mode/javascript/javascript.js");
-  require("codemirror/mode/htmlmixed/htmlmixed.js");
-  require("codemirror/mode/python/python.js");
-  require("codemirror/mode/clike/clike.js");
-}
-/**----------Props--------
- * None
- */
-
 class TextEditor extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      codeMirrorInstance: null,
+      editorInstance: null,
```

Comment:
purpose of this parameter?

Reply:
Ngl i have no idea. it was in the code mirror version. once it mounts, it says it has been made